msprime/demography.py

```diff
@@ -3186,10 +3181,14 @@ def _as_lineage_movements(self):
 @dataclasses.dataclass
 class ActivatePopulationEvent(DemographicEvent):
     """
-    :param int population: The ID of the population to activate.
+    A population activation event, which changes the state of the population
+    from inactive to active at the given time.
     """
 
     population: int | str
+    """
+    :param int population: The ID of the population to activate.
```

Comment:
We don't want the :param: notation - see the example [here](https://github.com/tskit-dev/msprime/blob/975710471cd5efd26efb00eda4dd1c66e9811a71/msprime/demography.py#L88) for the recommended way to document dataclass attributes.

Reply:
The docstring from the latest commit has just "The ID of the population to activate.", and I removed the :param: notation. Not sure why it wasn't updated here.